core/pom.xml

```diff
@@ -150,6 +150,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.python</groupId>
+            <artifactId>jython</artifactId>
+            <version>2.7.0</version>
```

Comment:
I believe that as long as the archives are on the CLASSPATH this is possible, and executing `import whatever` will work, and have seen examples, but I have not tested this.